src/main/java/com/teragrep/pth_06/planner/walker/conditions/ElementCondition.java

```diff
@@ -92,6 +93,8 @@ public Condition condition() {
                 QueryCondition host = new HostCondition(value, operation, config.streamQuery());
                 condition = host.condition();
                 break;
+            default:
```

Comment:
I think this function could use some logic cleanup; but that is for another issue.